### DISCUSSION TITLE
Tahoma2d add more option for gradients

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1296] [Ideas]
1-instead of using the nodes for gradients there can be a tool for gradients or a way to select gradients/color for gradients (I know there's a vector brush but maybe there can be one for raster) or try the Adobe animate cc method where you can use the paint tool and the gradient tool

2-add more colors for gradients like you can have more then two colors for the gradients

**Comment by animatewithdermot** [comment]
Adding more than 2 cols for gradients (as in Flash, which had this over 20 years ago, would be great). Unless I'm missing something we're still on two.
